dspace/config/dspace.cfg

```diff
@@ -1585,6 +1585,17 @@ solr-database-resync.time-until-reindex = 600000
 # Keep in mind, changing the schedule requires rebooting your servlet container, e.g. Tomcat.
 solr-database-resync.cron = 0 15 2 * * ?
 
+#----------------------------------------------------------#
+#----------PROCESS CONFIGURATION------------#
+#----------------------------------------------------------#
+# Configuration to enable saving script via CLI in DB
+# default = false
+#process.save-enable = false
+# !!! if process.save is enabled, make sure that this field has a correct EPerson's email
+#process.eperson=
```

Comment:
Just a quick comment.  I understand why this is necessary, as it appears an [EPerson is *required* to create a Process](https://github.com/DSpace/DSpace/blob/main/dspace-api/src/main/java/org/dspace/scripts/Process.java#L54-L56).  However, I still find this a bit clunky.  I'm worried that it's misleading that CLI processes could appear under a EPerson who didn't run the process.

But, I admit, I don't have a better "quick solution". The only way to make this work without a config would be to either:
1. Change Process to allow for a null EPerson
2. Or, require all Scripts to specify an EPerson (via a flag) when run from CLI.

Neither of those seem like quick solutions though.  But, I wanted to mention them here in case it prompts ideas from other developers.

Reply:
The first one seems more efficient than "force" users to set a flag everytime, especially those that have a lot of crontabs to manage Dspace process. I'll see how this null Eperson field impact in Dspace Process, if you agree.

Reply:
I'm OK with investigating the effort to allow for a `null` EPerson in a Process.  But, I'll admit, I have not dug into this concept deeply at all, so I don't know whether it will have negative side effects.

Reply:
Hi! I'm back. After a deep look, I've seen that setting `null` EPerson doesn't seem to have side effects. 
The only place where it might causes a NPE is [here](https://github.com/DSpace/DSpace/blob/9cbfad4df837d17fa99147d99ea285a0f2f47097/dspace-server-webapp/src/main/java/org/dspace/app/rest/converter/ProcessConverter.java#L42l) because of `process.getEPerson().getID()`.

Reply:
@Bezkup : In that case, if you want to update this PR to use a `null` EPerson, and fix any code where an NPE might occur, that seems reasonable.  

I would also recommend "documenting" that Eperson may be null.  There's two logical way to do that.  One is to add tests (or ITs) which prove that a null EPerson is "valid" (i.e. doesn't throw errors)... that might be the best approach anyhow in proving this enhancement "works".   The other would be to update JavaDocs to document this (if it's too difficult to find a place to add a test where Eperson is set to null).

Overall, this sounds like the approach we'd want for this PR if you have time to clean this up in the near future.  

As an FYI, we are in the review process for 8.0 PRs. So if you can find time to update this in the next week or two, it'd allow us to consider this for inclusion. If you don't have time soon, we can do our best to move this forward when you have time...but whether it gets into 8.0 may depend on whether we still have time to safely merge it.  Thanks!

Reply:
@tdonohue Nice! I think I'm done (waiting for your feedbacks) :) 
I added a test case for create a `Process` without `EPerson` in `ProcessIT` class, hope it's the correct class to put it.